lms/templates/instructor/instructor_dashboard_2/data_download.html

```diff
@@ -39,6 +39,19 @@
       <p>${_("Click to generate a CSV file of all proctored exam results in this course.")}</p>
       <p><input type="button" name="proctored-exam-results-report" value="${_("Generate Proctored Exam Results Report")}" data-endpoint="${ section_data['list_proctored_results_url'] }"/></p>
     %endif
+
+    <p>${_("Enter a problem URL, then click to generate a CSV file that lists all student answers to a given problem.")}</p>
```

Comment:
OK, so having used this feature I'm going to suggest a bit of rewording here (sorry for the back and forth). I think "URL" may be confusing, and result in people trying to copy the literal url from the address bar. Maybe,

"Enter a problem location (obtained from the Staff Debug Info for a given problem), then click..."

@lamagnifica thoughts on exact wording? (Tim, I'd wait until Alison weighs in before making a change)